docker/test/stress/run.sh

```diff
@@ -127,23 +127,24 @@ EOL
 
 function stop()
 {
-    local max_tries=""
-    if [ -n "$1" ]
-    then
-        max_tries="--max-tries $1"
-    fi
```

Comment:
Initially there was a check, that had been removed in dc52a40c045a94c7e3341daa467dff0d395c375d, and that's why it started to fail.